components/FilterableNumbersList.js

```diff
@@ -32,6 +32,7 @@ class _FilterableNumbersList extends React.Component {
     return (
       <section>
         <DebounceInput
+          className="search-box"
```

Comment:
I'm not a big fan of the fact that we need to add class names to enable interactions with certain parts of these elements within the cypress tests 😅 My reasoning is that the name becomes duplicated across the tests and the implementation, which is a potential source of drift

It would be nice to be able to reference components directly, but I didn't find something within the API that would enable me to do it. I'm imagining something like the [Component Constructors feature of Enzyme](https://airbnb.io/enzyme/docs/api/ReactWrapper/find.html), though I think that would be tough since it's React specific API 😄 #foodforthought

Reply:
we have an entire guide on this http://on.cypress.io/best-practices#Selecting-Elements